### DISCUSSION TITLE
Compatibility with the Next 13 `/app` directory

### DISCUSSION THREAD

**Discussion by 31i0t** [RyanClementsHax/storybook-addon-next#159] [Ideas]
Support for the new Next 13 `/app` directory! Currently, using any of [the new client hooks](https://beta.nextjs.org/docs/getting-started) breaks Storybook.

**Comment by RyanClementsHax** [comment]
For this, I'd recommend migrating to `@storybook/nextjs`. They have better support for the `/app` directory. I'm going to deprecate this addon anyway in favor of their version.

The migration should be easy since they used this addon as a base and I helped them develop it.

Let me know if you run into any issues.
